cocinaborcelle/src/components/cardComida.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import styles from "@/styles/CardComida.module.css";
+
+type CardComidaProps = {
+    title?: string;
+}
```

Comment:
```suggestion
import { Comida } from '@prisma/client';

type CardComidaProps = {
    comida: Comida
}
```

é melhor passa algo que é retornado da api para o componente, e aí ele recebe tudo que precisa para formar o card